polygon/rest/quotes.py

```diff
@@ -115,7 +115,7 @@ def get_real_time_currency_conversion(
         self,
         from_: str,
         to: str,
-        amount: float,
+        amount: Optional[float] = None,
```

Comment:
inorder for parameters to be used in get_params, they must have set defaults. This change allowed us to remove line 136-139

Reply:
This param isn't optional in the docs but apparently defaults to 100 :shrug:
![image](https://user-images.githubusercontent.com/43246297/207515481-59a6ec59-96e8-4867-b821-8e37faec19a2.png)